src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1047,7 +1047,7 @@ private TokenValidationResult ValidateToken(string token, JsonWebToken outerToke
                         else
                         {
                             currentConfiguration = validationParameters.ConfigurationManager.LastKnownGoodConfiguration;
-                            tokenValidationResult = decryptedJwt != null ? ValidateJWE(outerToken, decryptedJwt, validationParameters, currentConfiguration) : ValidateJWS(token, validationParameters, currentConfiguration); ;
+                            tokenValidationResult = decryptedJwt != null ? ValidateJWE(outerToken, decryptedJwt, validationParameters, currentConfiguration) : ValidateJWS(token, validationParameters, currentConfiguration); 
```

Comment:
>  [](http://example.com/codeflow?start=222&length=1)

supernit: trailing space

Reply:
and as long as you're touching this line splitting the ternary statement to three lines makes it more readable imo but that might be just a style thing.

Reply:
@TimHannMSFT future work that is planned right after this PR related to using System.Text.Json will clean this up.